SwiftLeeds/Network/URLSession.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  URLSession.swift
+//  SwiftLeeds
+//
+//  Created by Matthew Gallagher on 24/09/2023.
+//
+
+import Foundation
+
+public extension URLSession {
+    static var awaitConnectivity: URLSession = {
+        let configuration = URLSessionConfiguration.default
+        configuration.waitsForConnectivity = true
+        configuration.timeoutIntervalForRequest = 5 //60
+        configuration.timeoutIntervalForResource = 5 //60 * 30
```

Comment:
are the comments here placeholders of production values?

Reply:
Original timeout values, I'll get rid